lib/google_crawler/search/keyword_file.ex

```diff
@@ -0,0 +1,37 @@
+defmodule GoogleCrawler.Search.KeywordFile do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  embedded_schema do
```

Comment:
It's a great idea to use `embedded_schema`. I just read about it last weekend. It's the closest to implement some kind of form object 💪